src/App.js

```diff
@@ -7,19 +9,31 @@ import NewPetForm from './components/NewPetForm';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
 
-import { pets } from './data/pets.json';
-
 class App extends Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      petList: pets,
+      petList: [],
       currentPet: undefined,
-      originalPets: pets,
+      originalPets: [],
     };
   }
 
+  componentDidMount () {
+    axios.get('http://localhost:2999/pets')
+      .then((response) => {
+        this.setState({
+          petList: response.data,
+          originalPets: response.data,
```

Comment:
I know you didn't add it but what do we use `originalPets` for that we couldn't just use `petList` for?  (Since it looks like you're setting `originalPets` every time you set `petList`.)

Reply:
It's used for filtering.

Reply:
I did change the name however to fullList

Reply:
I was actually just reading about how this is an antipattern according to react documentation.
It's storing more data in state than needs to be stored, cause it could just be storing the filter term in state, right?

Reply:
That said, I know it was already like this so maybe that's out of scope for this change.
The thing I read was step 3 here: https://reactjs.org/docs/thinking-in-react.html

Reply:
^ This would involve lifting the state out of search bar.

Reply:
You're right in that we should instead be lifting state out of the search bar and then using that term to filter the prop passed into the PetList.

Reply:
I've updated it to lift the search form's state up.

Reply:
Cool :)